packages/flutter_tools/templates/module/common/.gitignore.tmpl

```diff
@@ -9,6 +9,8 @@
 .sconsign.dblite
 .svn/
 
+migrate_working_dir/
```

Comment:
does the tool also support Flutter modules?

Reply:
No. Not yet.

Reply:
But if the future if/when we do, it would be nice to have this already in the .gitignore to allow for better backwards compatibility.